jcl/src/java.base/share/classes/openj9/internal/foreign/abi/InternalUpcallHandler.java

```diff
@@ -108,10 +160,26 @@ public InternalUpcallHandler(MethodHandle target, MethodType mt, FunctionDescrip
 		TypeLayoutCheckHelper.checkIfValidLayoutAndType(mt, argLayoutArray, realReturnLayout);
 		/*[ENDIF] JAVA_SPEC_VERSION == 17 */
 
+		try {
```

Comment:
Please fix the javadoc for this and other constructors: `@return` is not appropriate.

Reply:
Removed everywhere for constructor as suggested.